include/inspect.h

```diff
@@ -508,6 +508,15 @@ bool inspect_badfuncs(struct rpminspect *ri);
  */
 bool inspect_runpath(struct rpminspect *ri);
 
+/**
+ * @brief Main driver for the 'unicode' inspection.
+ *
+ *
+ * @param ri Pointer to the struct rpminspect for the program.
```

Comment:
To the struct rpminspect uses? It feels like it's missing a word.

Reply:
This is a placeholder for the Doxygen work I plan to do for the library.  That's why it says '@param' and has a blank spot in the middle.  I just copy one from above and change it and always remind myself that I will come back and complete the Doxygen work.

In this context it's 'struct rpminspect' as in the data type.  Again, I will clean this up with the mega doxygen work coming soon.